src/foundry/client/apps/sidebar/package-configuration.d.ts

```diff
@@ -0,0 +1,97 @@
+export {};
+
+/**
+ * An application for configuring data across all installed and active packages.
+ */
+declare global {
+  abstract class PackageConfiguration<ConcreteCategory extends PackageConfiguration.Category> extends FormApplication<
+    PackageConfiguration.Options,
```

Comment:
What about making `Options` a generic parameter, so that overriding classes can extend it? Same for `Data`.